polylabel.js

```diff
@@ -73,7 +77,9 @@ function polylabel(polygon, precision, debug) {
         console.log('best distance: ' + bestCell.d);
     }
 
-    return [bestCell.x, bestCell.y];
+    var poleOfInaccessibility = [bestCell.x, bestCell.y];
+    poleOfInaccessibility.distance = bestCell.d || 0;
```

Comment:
Note: the `|| 0` here is intended to replace occurances of `-0` with `0`.

Reply:
we could clarify this earlier in [polylabel.js#L110](https://github.com/mapbox/polylabel/blob/master/polylabel.js#L110):
```
return minDistSq && (inside ? 1 : -1) * Math.sqrt(minDistSq);
```

Reply:
Ah I see, maybe something like `return minDistSq ? (inside ? 1 : -1) * Math.sqrt(minDistSq): 0;`

Reply:
Or maybe more explicit:

`return minDistSq !== 0 ? (inside ? 1 : -1) * Math.sqrt(minDistSq) : 0;`

Reply:
Or this variant might be most readable:

`return minDistSq === 0 ? 0 : (inside ? 1 : -1) * Math.sqrt(minDistSq);`